src/services/recim/series.ts

```diff
@@ -107,8 +116,10 @@ const getSeriesSchedule = (seriesID: number): Promise<SeriesSchedule> =>
 const putSeriesSchedule = (schedule: UploadSeriesSchedule): Promise<SeriesSchedule> =>
   http.put(`recim/series/schedule`, schedule);
 
-const scheduleSeriesMatches = (seriesID: number): Promise<Match[]> =>
-  http.post(`recim/series/${seriesID}/autoschedule`, {});
+const scheduleSeriesMatches = (
+  seriesID: number,
+  params: AutoScheduleParameters | null,
```

Comment:
Redundant since `AutoScheduleParameters` can be `null`.
```suggestion
  params: AutoScheduleParameters,
```